eddy/core/items/edges/common/base.py

```diff
@@ -427,6 +428,15 @@ def itemChange(self, change, value):
             self.updateEdge(selected=value)
         return super().itemChange(change, value)
 
+    def mouseDoubleClickEvent(self, mouseEvent):
+        """
+        Add breakpoint to create right angle when the mouse is double clicked on the text item.
+        :type mouseEvent: QGraphicsSceneMouseEvent
+        """
+        if not self.breakpoints and self.source.pos().x() != self.target.pos().x() and self.source.pos().y() != self.target.pos().y():
```

Comment:
Puoi usare direttamente un confronto del tipo:
```python
self.source.pos() != self.target.pos()
```
in quanto QPoint(F) hanno gli operatori speciali